### PR TITLE
Update bulk-import-external-contacts.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated Steps 3 with the correct Cmdlet. Tested and verified

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9120

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/bulk-import-external-contacts.md [hunk 1]
```diff
@@ -102,12 +102,7 @@ After you run the command in Step 2, the external contacts are created, but they
 3. Run the following two commands to add the other properties from the CSV file to the external contacts that you created in Step 2.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/bulk-import-external-contacts.md]:
```suggestion
3. Run the following command to add the other properties from the CSV file to the external contacts that you created in Step 2.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@v-tophillips Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by v-tophillips — APPROVED** [review]

**Pull request merged**